core/tests/protractor_utils/general.js

```diff
@@ -41,6 +41,16 @@ var checkForConsoleErrors = function(errorsToIgnore) {
         return !(browserLog.message.includes(' Slow network is detected.'));
       });
     }
+
+    // Since we are in the process of upgrading CKeditor-4 to Ckeditor-5,
+    // we are observing consoles errors for duplicated modules for more details,
```

Comment:
I think you're still missing `(` before `for more details...` and `)` after `refer to ...1821`. Please see my original comment https://github.com/oppia/oppia/pull/7184#discussion_r306641220 carefully.

Reply:
Done PTAL!